pkg/function/scale_workload.go

```diff
@@ -167,8 +167,8 @@ func (s *scaleWorkloadFunc) setArgs(tp param.TemplateParams, args map[string]int
 	case int64:
 		replicas = int32(val)
 	case string:
-		var v int
-		if v, err = strconv.Atoi(val); err != nil {
+		v, err := strconv.ParseInt(val, 10, 32)
```

Comment:
In the older version if `val` (`string` type) has an int value that is beyond the range of `int32`, even though call `strconv.Atoi(val)` would succeed, the next of `replicas = int32(v)` would result into unexpected result because the val is beyond what can be accommodated into int32 type.
This change fixes that.